mitodl/mitxpro
# Tasawer/242/site notifications

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #242 

#### What's this PR do?
Add feature for site notifications.

#### How should this be manually tested?

- Add new notification through `cms/snippets/cms/sitenotification/`
- Visit any page, you should see notification.
- Notification should disappear for the rest of the session. 


#### Any background context you want to provide?
RFC: https://github.com/mitodl/mitxpro/pull/306

#### Screenshots (if appropriate)
 **Wagtail:**
<img width="1042" alt="Screen Shot 2019-05-20 at 4 01 14 PM" src="https://user-images.githubusercontent.com/4245618/58016962-80180480-7b18-11e9-8cf7-cf8c57cdffa2.png">


**Rendered:**
<img width="1276" alt="Screen Shot 2019-05-20 at 4 00 38 PM" src="https://user-images.githubusercontent.com/4245618/58016913-64acf980-7b18-11e9-92cb-5931b6938f1b.png">

**Out of scope:**
- date range between which the content should be displayed
- ensure that only one site wide notification can be displayed at once (@pdpinch I'm guessing we want this to avoid overwhelming the user?)
